examples/vello_editor/src/text.rs

```diff
@@ -53,6 +53,32 @@ impl Editor {
             WindowEvent::ModifiersChanged(modifiers) => {
                 self.modifiers = Some(modifiers);
             }
+            WindowEvent::Ime(ime) => {
+                self.editor.transact(
+                    &mut self.font_cx,
+                    &mut self.layout_cx,
+                    match ime {
+                        Ime::Enabled => vec![],
+                        Ime::Disabled => vec![
+                            PlainEditorOp::SetCompose("".into(), None),
+                            PlainEditorOp::CommitCompose,
+                        ],
+                        // Winit on some platforms delivers an empty Preedit after Commit
+                        // so don't lock into compose when preedit is empty.
+                        Ime::Preedit(text, None) if text.is_empty() => vec![
+                            PlainEditorOp::SetCompose("".into(), None),
```

Comment:
A dedicated `ClearCompose` Op for setting to empty might be nice.

Reply:
That's fair, and definitely the ergonomic choice when moving to methods instead of ops.